diff --git a/ublox/ublox_gps/config/zed_f9p.yaml b/ublox/ublox_gps/config/zed_f9p.yaml
index 9fd45c7..835ef29 100644
--- a/ublox/ublox_gps/config/zed_f9p.yaml
+++ b/ublox/ublox_gps/config/zed_f9p.yaml
@@ -5,7 +5,7 @@ ublox_gps_node:
     device: /dev/ttyACM0
     frame_id: gps
     uart1:
-      baudrate: 9600
+      baudrate: 230400
     # TMODE3 Config
     tmode3: 1                   # Survey-In Mode
     sv_in:
diff --git a/ublox/ublox_gps/launch/ublox_gps_node-launch.py b/ublox/ublox_gps/launch/ublox_gps_node-launch.py
index 9a8d9b2..5fefdf9 100644
--- a/ublox/ublox_gps/launch/ublox_gps_node-launch.py
+++ b/ublox/ublox_gps/launch/ublox_gps_node-launch.py
@@ -43,7 +43,7 @@ def generate_launch_description():
     config_directory = os.path.join(
         ament_index_python.packages.get_package_share_directory('ublox_gps'),
         'config')
-    params = os.path.join(config_directory, 'c94_m8p_rover.yaml')
+    params = os.path.join(config_directory, 'zed_f9p.yaml')
     ublox_gps_node = launch_ros.actions.Node(package='ublox_gps',
                                              executable='ublox_gps_node',
                                              output='both',
